packages/commercetools/theme/components/Header/MobileMenu.vue

```diff
@@ -0,0 +1,121 @@
+<template>
```

Comment:
this file is (mostly) copy-pasted - can't you just implement logic once if this is repetitive?